redisinsight/ui/src/components/main-router/config.ts

```diff
@@ -0,0 +1,14 @@
+import { isArray } from 'lodash'
+import { IRoute } from 'uiSrc/constants'
+
+export const LAZY_LOAD = window.riConfig?.app?.lazyLoad
+
+export const RI_ROUTES_EXCLUDED_BY_ENV = window.riConfig?.app?.routesExcludedByEnv
+
+export const getRouteIncludedByEnv = (routeDefinition: IRoute | IRoute[]) => {
+  if (RI_ROUTES_EXCLUDED_BY_ENV) {
```

Comment:
did you check that the build is smaller with this env variable?

Reply:
The build size doesn't seem to be changing regardless of whether or not this variable is set